source/kraken/worker.cpp

```diff
@@ -692,32 +783,33 @@ pbnavitia::Response Worker::dispatch(const pbnavitia::Request& request) {
-        case pbnavitia::ISOCHRONE:
-        case pbnavitia::NMPLANNER:
-        case pbnavitia::pt_planner:
-        case pbnavitia::PLANNER: response = journeys(request.journeys(), request.requested_api(),
-                                                     current_datetime); break;
-        case pbnavitia::places_nearby: response = proximity_list(request.places_nearby(), current_datetime); break;
-        case pbnavitia::PTREFERENTIAL: response = pt_ref(request.ptref(), current_datetime); break;
-        case pbnavitia::traffic_reports : response = traffic_reports(request.traffic_reports(),
-                                                                     current_datetime); break;
-        case pbnavitia::calendars : response = calendars(request.calendars(), current_datetime); break;
-        case pbnavitia::place_code : response = place_code(request.place_code()); break;
-        case pbnavitia::nearest_stop_points : response = nearest_stop_points(request.nearest_stop_points()); break;
-        default:
-            LOG4CPLUS_WARN(logger, "Unknown API : " + API_Name(request.requested_api()));
-            fill_pb_error(pbnavitia::Error::unknown_api, "Unknown API", response.mutable_error());
-            break;
+    case pbnavitia::places: response = autocomplete(request.places(), current_datetime); break;
+    case pbnavitia::pt_objects: response = pt_object(request.pt_objects(), current_datetime); break;
+    case pbnavitia::place_uri: response = place_uri(request.place_uri(), current_datetime); break;
+    case pbnavitia::ROUTE_SCHEDULES:
+    case pbnavitia::NEXT_DEPARTURES:
+    case pbnavitia::NEXT_ARRIVALS:
+    case pbnavitia::PREVIOUS_DEPARTURES:
+    case pbnavitia::PREVIOUS_ARRIVALS:
+    case pbnavitia::DEPARTURE_BOARDS:
+        response = next_stop_times(request.next_stop_times(), request.requested_api(), current_datetime); break;
+    case pbnavitia::ISOCHRONE:
+    case pbnavitia::NMPLANNER:
+    case pbnavitia::pt_planner:
+    case pbnavitia::PLANNER: response = journeys(request.journeys(), request.requested_api(),
+                                                 current_datetime); break;
+    case pbnavitia::places_nearby: response = proximity_list(request.places_nearby(), current_datetime); break;
+    case pbnavitia::PTREFERENTIAL: response = pt_ref(request.ptref(), current_datetime); break;
+    case pbnavitia::traffic_reports : response = traffic_reports(request.traffic_reports(),
+                                                                 current_datetime); break;
+    case pbnavitia::calendars : response = calendars(request.calendars(), current_datetime); break;
+    case pbnavitia::place_code : response = place_code(request.place_code()); break;
+    case pbnavitia::nearest_stop_points : response = nearest_stop_points(request.nearest_stop_points()); break;
+    case pbnavitia::graphical_isochron : response = graphical_isochron(request.isochron(), current_datetime); break;
```

Comment:
aha,  just be careful when you commit, you actually modified only one line here, but the whole block is touched, the guilty may be your IDE :)